internal/sysfs/dirfs_test.go

```diff
@@ -758,7 +758,7 @@ func TestDirFS_Truncate(t *testing.T) {
 
 			name := "truncate"
 			realPath := path.Join(tmpDir, name)
-			require.NoError(t, os.WriteFile(realPath, content, 0o0600))
+			require.NoError(t, os.WriteFile(realPath, content, 0o0666))
```

Comment:
in windows these always turn to world write, we in other places use 0o0666 because of this. I noticed a couple places we didn't.